packages/foam-core/src/core.ts

```diff
@@ -0,0 +1,89 @@
+import { Graph, Edge } from 'graphlib'
+
+
+type ID = string
+
+export interface Link {
+  from: ID
+  to: ID
+  text: string
+}
+
+export interface NoteLink {
+  to: ID
+  text: string
+}
```

Comment:
Maybe add fields from this? https://code.visualstudio.com/api/references/vscode-api#Position

Reply:
yeah this could be interesting, but I want to better understand the use case (do we need to update the info on every edit, or only on save? depends on what we need the info for I think)